### PR TITLE
Give only JWT for frontend callback

### PR DESCRIPTION
**Pull request by dmk**

### CODE REVIEW

**Comment by yivo** [comment]
@dkkoval JWT tokens must be provided by barong, not peatio!

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -33,9 +33,7 @@ def auth_hash
   def redirect_on_successful_sign_in
     "#{params[:provider].to_s.gsub(/(?:_|oauth2)+\z/i, '').upcase}_OAUTH2_REDIRECT_URL".tap do |key|
       if ENV[key] && params[:provider].to_s == 'barong'
-        auth_data = auth_hash['credentials']
-        auth_data['full_name'] = @member.full_name
-        redirect_to "#{ENV[key]}?#{auth_data.to_query}"
+          redirect_to "#{ENV[key]}?token=#{auth_hash['credentials']['token']}"
```
Comment (yivo) [inline app/controllers/sessions_controller.rb:36]: @dkkoval Can you reformat this and use CGI.escape?

**Review by yivo — CHANGES_REQUESTED** [review]

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (yivo) [reply app/controllers/sessions_controller.rb:36]: for token

**Comment by dmk** [comment]
@yivo let's close it since we're going to get JWT from barong

**Comment by yivo** [comment]
@dkkoval if need it urgently we may keep this in peatio

**Pull request closed without merging**